deploy/olm-catalog/performance-addon-operator/4.4.0/performance.openshift.io_performanceprofiles_crd.yaml

```diff
@@ -0,0 +1,162 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: performanceprofiles.performance.openshift.io
+spec:
+  group: performance.openshift.io
+  names:
+    kind: PerformanceProfile
+    listKind: PerformanceProfileList
+    plural: performanceprofiles
+    singular: performanceprofile
+  scope: Cluster
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: PerformanceProfile is the Schema for the performanceprofiles API.
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: PerformanceProfileSpec defines the desired state of PerformanceProfile.
+          properties:
+            cpu:
```

Comment:
this is missing `additionalKernelArgs`?

Reply:
ah, i got this crd from d/s repo - it does not have KernelArgs for some reason..

will add it